frontend/javascripts/admin/job/job_list_view.tsx

```diff
@@ -137,7 +137,9 @@ function JobListView() {
   const [isLoading, setIsLoading] = useState(true);
   const [jobs, setJobs] = useState<APIJob[]>([]);
   const [searchQuery, setSearchQuery] = useState("");
-  const isCurrentUserSuperUser = useSelector((state: OxalisState) => state.activeUser?.isSuperUser);
+  const isCurrentUserSuperUser = useSelector(
```

Comment:
useWkSelector?

Reply:
Good catch. I replace another batch of `useSelector` calls.

Reply:
great, however, I think some imports are missing now (see ci output).